openpgp/internal/ecc/ed448.go

```diff
@@ -53,6 +53,19 @@ func (c *ed448) UnmarshalByteSecret(s []byte) (d []byte) {
 	return s[1:]
 }
 
+func (c *ed448) UnmarshalSignature(r, s []byte) (sig []byte) {
+	// Only R is used, in prefixed native format draft-ietf-openpgp-crypto-refresh-06#section-9.2.1
+	if len(r) != ed448lib.SignatureSize + 1 {
+		return nil
+	}
+
+	return r[1:]
+}
+
+func (c *ed448) MarshalSignature(sig []byte) (r, s []byte) {
+	return append([]byte{0x40}, sig...), nil
```

Comment:
This seems to work but it feels a bit accidental that using `NewMPI(nil)` works, and if we ever call `Bytes()` on it for some reason and try to use those, we might panic. So I would return `[]byte{}` for `s` instead, and maybe add a comment that `S` is supposed to be an MPI with value 0.